source/includes/_node.md

```diff
@@ -2,106 +2,216 @@
 
 ## JSON-RPC Requests 
 
-```shell
-
-```
-
-```javascript
-
-```
+Route for JSON-RPC requests, most of which mimic the bitcoind RPC calls completely.
 
-> The above command returns JSON structured like this:
+### HTTP Request
+`POST /`
```

Comment:
I think this should have a full example of doing a post with RPC, similar to what is in the wiki now (like below) but with the full sample request and response. This might help with the styling a little because I think this without any code sample is messing up the spacing a bit.

```
Request Body: {"method":"getblockchaininfo","params":[]}
Response Body: {"id":...,"result":...}
```